### PR TITLE
Refresh network config CAP

### PR DESCRIPTION
**Pull request by sisuresh**

### CODE REVIEW

File: core/cap-0046-09.md [hunk 1]
```diff
@@ -86,18 +86,21 @@ is only being partially updated. This is done to simplify the upgrade format
...
+to exist on ledger in a `ContractDataEntry`, and the validators will look it up
+using the `ConfigUpgradeSetKey`. This means that a WASM contract will need to be
+used to write the proposed upgrade. The specs of the entry written is as follows -
+- It should be a `TEMPORARY` entry, so make sure it's bumped to live long enough for the upgrade.
+- The `SCVal` `key` should be of type `SCV_BYTES`, where the value is the SHA-256 hash of the `ConfigUpgradeSet`.
+- The `SCVal` `val` should also be of type `SCV_BYTES`, where the value is the
+  serialized XDR of the `ConfigUpgradeSet`.
+  
+Once the proposed upgrade is setup, you'll need to share the
+`ConfigUpgradeSetKey` with the other validators so they can vote for it.
```
Comment (dmkozh) [inline core/cap-0046-09.md:103]: I think we should document somewhere that core validates the nominated entry by verifying that the value hash equals to the key hash.

File: core/cap-0046-09.md [hunk 2]
```diff
@@ -110,28 +113,27 @@ have exactly the same upgrade set).
...
+To allow increasing `ledgerMaxTxCount` from 0, we introduce the ability to set
+the value using the traditional upgrade mechanism. There's a new `LedgerUpgrade`
+type `LEDGER_UPGRADE_MAX_SOROBAN_TX_SET_SIZE`, which contains a `uint32
+newMaxSorobanTxSetSize` to update
+`ConfigSettingContractExecutionLanesV0::ledgerMaxTxCount`.
 
-If there are ever multiple valid `ConfigUpgradeSet`s that validator can accept,
-the one that has the most entries should be picked. Ties are resolved in favor
-of the lowest hash value.
+#### Minimum values
```
Comment (dmkozh) [inline core/cap-0046-09.md:131]: I think we should either document the minimum values here or in some other CAPs. Probably here we should just say that CAPs that introduce settings should also define the minimum acceptable values.

File: core/cap-0046-09.md [hunk 3]
```diff
@@ -110,28 +113,27 @@ have exactly the same upgrade set).
 Individual config settings are validated in the way defined by the CAPs
 introducing them.
 
-#### `ConfigUpgradeSet` acceptance
+#### Disabling Soroban Transactions
```
Comment (dmkozh) [inline core/cap-0046-09.md:116]: Why did you remove the 'acceptance' section? I don't think it has changed much.

Comment (sisuresh) [reply core/cap-0046-09.md:116]: That section was mainly about validators accepting a ConfigSettingEntry through some mechanism in consensus  in consensus. I didn't think it was necessary (or accurate) now that the entry is expected to live on ledger.

Comment (dmkozh) [reply core/cap-0046-09.md:116]: Ok, I think we still need to mention somewhere that validators can only vote for an upgrade set key that they have scheduled (maybe not as a separate section).

Comment (sisuresh) [reply core/cap-0046-09.md:116]: I mention this under the `Configuration upgrade mechanism` section. Do you think that's sufficient?
```
The actual `ConfigUpgradeSet` will need
to exist on ledger in a `ContractDataEntry`, and the validators will look it up
using the `ConfigUpgradeSetKey`.
```

File: core/cap-0046-09.md (hunk 1 above)
Comment (sisuresh) [reply core/cap-0046-09.md]: Done

**Review by dmkozh — DISMISSED** [review]

File: core/cap-0046-09.md (hunk 3 above)
Comment (sisuresh) [reply core/cap-0046-09.md:121]: Ah ok. Updated in the last commit.

**Review by dmkozh — APPROVED** [review]
